tests/test_grid3d/test_grid_property.py

```diff
@@ -697,3 +696,58 @@ def test_gridprop_export_bgrdecl_double(tmp_path, grid):
 
     with open(tmp_path / "actnum.DATA", "rb") as fh:
         assert b"DOUB" in fh.read()
+
+
+@pytest.mark.parametrize(
+    "simcase, props, expected_avg, expected_stddev",
+    [
+        (
+            pytest.param(
+                "E100_BO",
+                ["SOIL", "PRESSURE"],
+                [0.0119888, 270.03588],
+                [0.01306, 0.06424],
+                id="E100 blackoil",
+            )
+        ),
+        (
+            pytest.param(
+                "E300_BO",
+                ["SOIL", "PRESSURE"],
+                [0.0119888, 270.03588],
+                [0.013056, 0.0642213],
+                id="E300 blackoil",
+            )
+        ),
+        (
+            pytest.param(
+                "E300_COMP",
+                ["SOIL", "PRESSURE"],
+                [0.00945095, 400.720811],
+                [0.011486, 0.042726],
+                id="E300 composite",
+            )
+        ),
+    ],
+)
+def test_simpleb8_case(simcase, props, expected_avg, expected_stddev):
+    """Test a very small case that is actually ran through E100, E300, IX."""
+
+    folder = TPATH / "3dgrids" / "simpleb8"
+
+    grd = xtgeo.grid_from_file(folder / str(simcase + ".EGRID"), fformat="egrid")
+
+    assert grd.dimensions == (4, 2, 3)
+
+    for inum, prop in enumerate(props):
+        gprop = xtgeo.gridproperty_from_file(
+            folder / str(simcase + ".UNRST"),
+            name=prop,
+            fformat="unrst",
+            date="20220101",
+            grid=grd,
+        )
+
+        assert gprop.values.mean() == pytest.approx(expected_avg[inum], rel=0.001)
+        # print(prop, gprop.values.std())
```

Comment:
Leftover from creating the snapshot. This is the reason I would prefer to have it as a snapshot, because then we are transparent that it is in fact a snapshot.